vector/src/main/java/im/vector/app/features/onboarding/OnboardingViewModel.kt

```diff
@@ -441,6 +457,11 @@ class OnboardingViewModel @AssistedInject constructor(
         }
     }
 
+    private fun handleUpdateUseCase() {
+        // TODO act on the use case selection
```

Comment:
Yeah, maybe just store it in a data store for now, for future usage in the app?

Reply:
I was planning to store and apply to the post hog in the next PR to keep this PR a little smaller, but I'm also happy to do the local storing here to avoid the TODO

Reply:
Ok, it's fine to do it in another PR. About analytics you do not have the user consent yet (in the current impl)

Reply:
that's fine, we'll store selection locally and when consent is granted we'll retrieve the stored value and apply as an identity property in the analytics init